api/envoy/api/v2/discovery.proto

```diff
@@ -186,8 +196,8 @@ message DeltaDiscoveryResponse {
   // The version of the response data (used for debugging).
   string system_version_info = 1;
 
-  // The response resources. These are typed resources that match the type url
-  // in the DeltaDiscoveryRequest.
+  // The response resources. The Resource proto's Any field ('resource') should be
+  // of type type_url.
```

Comment:
I think this one is less clear than the original.

Reply:
Partially reverted, although I have changed "the type url in the DeltaDiscoveryRequest" to "the type_url field", since I don't think DeltaDiscoveryRequest is relevant here (given that there can be spontaneous updates).